packages/url_launcher/url_launcher_macos/pigeons/messages.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:pigeon/pigeon.dart';
+
+@ConfigurePigeon(PigeonOptions(
+  dartOut: 'lib/src/messages.g.dart',
+  swiftOut: 'macos/Classes/messages.g.swift',
+  copyrightHeader: 'pigeons/copyright.txt',
+))
+
+/// Possible error conditions for [UrlLauncherApi] calls.
+enum UrlLauncherError {
+  /// The URL could not be parsed as an NSURL.
+  invalidUrl,
```

Comment:
I did this as an enum even though there's only one value since it was more self-documenting this way, and would easily extend to other error cases if we find more in the future.